tests/utils.py

```diff
@@ -1,11 +1,14 @@
-import pytz
+try:
+    import zoneinfo
+except ImportError:
+    from backports import zoneinfo
 from dateutil.zoneinfo import get_zonefile_instance
 
 
 def make_full_tz_list():
     dateutil_zones = set(get_zonefile_instance().zones)
-    pytz_zones = set(pytz.all_timezones)
-    return dateutil_zones.union(pytz_zones)
+    zoneinfo_zones = set(zoneinfo.available_timezones())
```

Comment:
Does this have the proper coverage for all timezones? We primarily used pytz to get greater coverage of time zones for the sake of tests

Reply:
I just compared them and it does have the same coverage as pytz (Windows 11). However, from the [zoneinfo docs](https://docs.python.org/3/library/zoneinfo.html#data-sources):

> The zoneinfo module does not directly provide time zone data, and instead pulls time zone information from the system time zone database or the first-party PyPI package [tzdata](https://pypi.org/project/tzdata/), if available. Some systems, including notably Windows systems, do not have an IANA database available, and so for projects targeting cross-platform compatibility that require time zone data, it is recommended to declare a dependency on tzdata. If neither system data nor tzdata are available, all calls to [ZoneInfo](https://docs.python.org/3/library/zoneinfo.html#zoneinfo.ZoneInfo) will raise [ZoneInfoNotFoundError](https://docs.python.org/3/library/zoneinfo.html#zoneinfo.ZoneInfoNotFoundError).